.editorconfig

```diff
@@ -0,0 +1,15 @@
+root = true
+
+[*]
+end_of_line = lf
+charset = utf-8
+indent_size = 2
```

Comment:
@Foggalong It won't cause any thing unless you modify the file. Which should never happen, if the license should be changed the whole file should be replaced as i don't think we are going to ever write a license from scratch. 
The README file, doesn't use any indentation and i don't think we will have any complicated stuff to show on this repository.  
And, tbh i have been using 2 as an indent size for README files which looks really normal to me, as you never have something complicated there; it's almost as plain text file